1.5/Defs/ThingDefs_Misc/FroggeMask.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+    <ThingDef ParentName="HatMakeableBase">
+        <defName>Apparel_FroggeVisageMask</defName>

```

Comment:
Change to `MSS_FP_Apparel_FroggeVisageMask` to keep our defs prefixed and prevent collisions